.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,3 @@
+Please review [EIP-1](../EIPS/eip-1.md) for EIP guidelines.
```

Comment:
I don't see where this appear on your example repository. I just wanted to make sure the relative link worked.

Reply:
Why use relative link here and absolute link below?

Reply:
The original relative link below doesn't work. However, an absolute link is likely better anyways, so I will replace it.